README.md

```diff
@@ -1,77 +1,103 @@
+[lbry daemon](https://github.com/lbryio/lbry) bundled with a UI using
+[Electron](http://electron.atom.io/).
 
-![App Screenshot](https://lbry.io/img/lbry-ui.png)
+![App screenshot](https://lbry.io/img/lbry-ui.png)
 
 ## Installing
 
 We provide installers for Windows, macOS, and Debian-based Linux.
 
-| | Windows | macOS | Linux |
-| --- | --- | --- | --- |
-| Latest Stable Release | [Download](https://lbry.io/get/lbry.exe) | [Download](https://lbry.io/get/lbry.dmg) | [Download](https://lbry.io/get/lbry.deb) |
-| Latest Prerelease | [Download](https://lbry.io/get/lbry.pre.exe) | [Download](https://lbry.io/get/lbry.pre.dmg) | [Download](https://lbry.io/get/lbry.pre.deb) |
+|                       | Windows                                      | macOS                                        | Linux                                        |
+| --------------------- | -------------------------------------------- | -------------------------------------------- | -------------------------------------------- |
+| Latest Stable Release | [Download](https://lbry.io/get/lbry.exe)     | [Download](https://lbry.io/get/lbry.dmg)     | [Download](https://lbry.io/get/lbry.deb)     |
+| Latest Prerelease     | [Download](https://lbry.io/get/lbry.pre.exe) | [Download](https://lbry.io/get/lbry.pre.dmg) | [Download](https://lbry.io/get/lbry.pre.deb) |
 
-
-Our [releases page](https://github.com/lbryio/lbry-app/releases/latest) also contains the latest release, pre-releases, and past builds.
+Our [releases page](https://github.com/lbryio/lbry-app/releases/latest) also contains the latest
+release, pre-releases, and past builds.
 
 To install from source or make changes to the application, continue reading below.
 
-## Development on Linux and macOS
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for
+development and testing purposes.
+
+### Prerequisites
+
+* [Git](https://git-scm.com/downloads)
+* [Node.js](https://nodejs.org/en/download/)
+* [Yarn](https://yarnpkg.com/en/docs/install)
+* `yarn --add-python-to-path install --global --production windows-build-tools` (Windows only)
 
 ### One-time Setup
 
 1. Clone this repo
 2. `DEPS=true ./build.sh`
 
-This will download and install the LBRY app and its dependencies, including [the LBRY daemon](https://github.com/lbryio/lbry) and command line utilities like `node` and `yarn`. The LBRY app requires Node >= 6; if you have an earlier version of Node installed and want to keep it, you can use [nvm](https://github.com/creationix/nvm) to switch back and forth.
+This will download and install the LBRY app and its dependencies, including
+[the LBRY daemon](https://github.com/lbryio/lbry) and command line utilities like `node` and `yarn`.
+The LBRY app requires Node >= 6; if you have an earlier version of Node installed and want to keep
+it, you can use [nvm](https://github.com/creationix/nvm) to switch back and forth.
 
-### Ongoing Development
```

Comment:
Why did you take out this section? I don't see any explanation for how people are supposed to do ongoing dev

Reply:
ah nvm i see that its in contributing.md, but I think its still worth mentioning in this doc

Reply:
I've added a "Running" section. Eventually, we will be able to run the app without HMR by using yarn start. For the meantime, it's only yarn dev.